pkg/subctl/cmd/show.go

```diff
@@ -98,3 +104,20 @@ func getClientConfigAndClusterName(rules *clientcmd.ClientConfigLoadingRules, ov
 
 	return restConfig{config: clientConfig, clusterName: *clusterName}, nil
 }
+
+func getSubmarinerResource(config *rest.Config) *v1alpha1.Submariner {
+	submarinerClient, err := submarinerclientset.NewForConfig(config)
+	exitOnError("Unable to get the Submariner client", err)
+
+	submariner, err := submarinerClient.SubmarinerV1alpha1().Submariners(OperatorNamespace).Get(submarinercr.SubmarinerName, v1.GetOptions{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			fmt.Println("Submariner is not installed")
```

Comment:
```suggestion
```

Reply:
I still think it makes sense to print the message here since we're detecting it here and it avoids duplication in like 5 other places. Plus we print error messages here so it seems consistent to also print this message. But I'm not going to belabor it if others feel strongly about not printing here.